### PR TITLE
WIP: azure: do not delete nodes from single LB backend pool

### PR DESCRIPTION
**Pull request by staebler**
Attempt to bypass new code in Azure legacy-cloud-provider that deletes nodes from loadbalancer backend pools.

### CODE REVIEW

**Comment by staebler** [comment]
e2e-azure [passed](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_kubernetes/498/pull-ci-openshift-kubernetes-master-e2e-azure/1341508510399074304).

**Comment by deads2k** [comment]
This I can do.  I'd like this PR opened in k/k and the commit message changed to "UPSTREAM: XXXXX" where XXXXX is the upstream PR number.

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go [hunk 1]
```diff
@@ -1125,6 +1125,10 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 				klog.V(10).Infof("reconcileLoadBalancer for service (%s)(%t): lb backendpool - found wanted backendpool. not adding anything", serviceName, wantLb)
 				foundBackendPool = true
 
+				if !az.useStandardLoadBalancer() || !az.EnableMultipleStandardLoadBalancers {
```
Comment (michaelgugino) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go:1128]: This says 'not useStandardLoadBalancer' is this intended?  What kind of load balancer are we using?  Do we use some that are standard and some that are not standard?

Comment (staebler) [reply staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go:1128]: We use a standard loadbalancer.

**Comment by staebler** [comment]
This is obsolete.
/close

**Pull request closed without merging**